### PR TITLE
bert minor fixes v2

### PR DESCRIPTION
**Pull request by vikmary**
- [x] fix default parameter values
- [x] do not save BackupVariables while doing ExponentialMovingAverage of weights
- [x] re-upload bert ner models without the Variables (0.6x size saved)

### CODE REVIEW

File: deeppavlov/dataset_readers/conll2003_reader.py [hunk 1]
```diff
@@ -13,9 +13,11 @@ def read(self,
              data_path: str,
              dataset_name: str = None,
              provide_pos: bool = False,
+             iob: bool = False,
```
Comment (yoptar) [inline deeppavlov/dataset_readers/conll2003_reader.py]: If you're adding a new positional argument, you should add it to the end to not break existing calls

File: deeppavlov/dataset_iterators/document_bert_ner_iterator.py [hunk 2]
```diff
@@ -99,7 +101,7 @@ def gen_batches(self, batch_size: int, data_type: str = 'train',
         if num_docs == 0:
             return
 
-        if data_type == 'train':
+        if False:# data_type == 'train':
```
Comment (yoptar) [inline deeppavlov/dataset_iterators/document_bert_ner_iterator.py]: Maybe just remove this block then?

File: deeppavlov/dataset_iterators/document_bert_ner_iterator.py [hunk 3]
```diff
@@ -49,19 +49,21 @@ class DocumentBertNerIterator(DataLearningIterator):
     def __init__(self,
                  data: Dict[str, List[Tuple[Any, Any]]],
                  bert_tokenizer_vocab_file: str,
+                 do_lower_case: bool = False,
```
Comment (mu-arkhipov) [inline deeppavlov/dataset_iterators/document_bert_ner_iterator.py]: Feels like a relief

Comment (vikmary) [reply deeppavlov/dataset_iterators/document_bert_ner_iterator.py]: You bet!￼￼

File: deeppavlov/dataset_iterators/document_bert_ner_iterator.py [hunk 4]
```diff
@@ -99,35 +101,26 @@ def gen_batches(self, batch_size: int, data_type: str = 'train',
...
+        # get all sentences from document
+        doc_chunks = [self.chunks_from_doc(doc) for doc_id, doc in doc_data]
+        if self.one_sample_per_doc:
+            samples = [next(chunk) for chunk in doc_chunks]
         else:
-            # get all sentences from document
-            samples = [s for doc_id, doc in doc_data for s in self.chunks_from_doc(doc)]
-            num_samples = len(samples)
+            samples = [s for chunk in doc_chunks for s in chunk]
+        print(f"{len(samples)} samples from {len(doc_data)} docs")
```
Comment (mu-arkhipov) [inline deeppavlov/dataset_iterators/document_bert_ner_iterator.py]: to logger

Comment (vikmary) [reply deeppavlov/dataset_iterators/document_bert_ner_iterator.py]: Removed!

**Pull request merged**